logux-vue-todomvc/public/index.html

```diff
@@ -0,0 +1,17 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
```

Comment:
I think we do not need IE support

Reply:
by Vue CLI, too 😅

Reply:
I am OK to keep JS config over JSON, but in this case, we can clean it.

It will be a good lesson to double-check everything which your boilerplate creates for you.

Reply:
Just noticed. You are right.